apps/docs-snippets/src/guide/cookbook/resubmitting-failed-transactions.test.ts

```diff
@@ -70,8 +70,11 @@ describe('Resubmitting Failed Transactions', () => {
           await wallet.sendTransaction(transactionRequest);
         }
       }
+
       // #endregion resubmitting-failed-transactions-3
-    }).rejects.toThrow('Transaction is not inserted. UTXO does not exist: 0x');
+    }).rejects.toThrow(
+      'Transaction input validation failed: UTXO (id: b2814e068053151c158a8c22c219d25ba5324d1ae6efe95fc82f06fcf928527b0000) does not exist'
+    );
```

Comment:
```suggestion
    }).rejects.toThrowError(/Transaction input validation failed: UTXO \(id: .*\) does not exist/);
```

Relates to changes [here](https://github.com/FuelLabs/fuel-core/pull/2216/files#diff-763e9a79947aa33db6000118408d6f1d8e639be46a2d6ff54939f097e7986aedR28) and [here](https://github.com/FuelLabs/fuel-core/blob/94fe64e876bbb651311f1e05a6ad77c54366ee65/crates/services/txpool_v2/src/error.rs#L130-L131)